weblab/core/processing.py

```diff
@@ -0,0 +1,11 @@
+from urllib.parse import urljoin
+
+from django.conf import settings
+
+def prepend_callback_base(url_path):
+    """Prepend CALLBACK_BASE_UR to url_path. If we are running in a subfolder this is striped from url_path."""
```

Comment:
How about this?
```suggestion
    """Prepend CALLBACK_BASE_URL to url_path.
    
    If we are running at a sub-URL (settings.FORCE_SCRIPT_NAME is set) then the prefix is stripped,
    since the callback is done on a private interface (to localhost) not via the public WebLab address.
    """
```